.github/workflows/lint.yml

```
@@ -0,0 +1,16 @@
+name: Lint
+
+on: pull_request
+
+jobs:
+  misc:
+    runs-on: [self-hosted, X64]
+    steps:
+      - uses: actions/checkout@v4
+      - name: spell check
+        run: |
+          # the python version in self-hosted runner is 3.7, which is not supported by the latest get-pip.py
+          python3 <(curl -s https://bootstrap.pypa.io/pip/3.7/get-pip.py)
+          pip install codespell==2.3.0
+          # ignore test files, go project names, binary files via `--skip` and special var/regex via `--ignore-words`
+          git grep --cached -l '' | xargs codespell  --skip 'fuzz/*,*_test.tmpl,testdata/*,*_test.go,go.mod,go.sum,*.gz' --ignore-words=.github/workflows/.ignore_words
```

I would simply use 

https://github.com/codespell-project/actions-codespell

So this


```suggestion
    steps:
      - uses: actions/checkout@v4
      - name: spell check
      - uses: codespell-project/actions-codespell@v2
```

 So with the .codespellrc file I'm suggesting in my other comment

---

@ccoVeille 
I would like to apply your suggestion, but there is a syntax error in it. Would you update it so I can commit the suggestion? Thanks!

---

oh indeed

---

is it OK now ?

---

ok

---

@spacewander ?